smslibrary/src/main/java/com/eis/communication/CommunicationHandler.java

```diff
@@ -9,7 +9,8 @@
 
     /**
      * Sends a single message in the channel, message content must be shorter than the maximum channel message size
-     * @param message message to be sent in the channel to a peer
+     *
+     * @param message the message to be sent in the channel to a peer
```

Comment:
You see, this is a correct edit to the code, I would have accepted it right away, but it's been edited with other 42 files and it's going to take a long time before it gets into the master code (if it's getting merged). You could have pulled changes like these in another request, and leave to this request the essential